SpectralCubeReprojectExample.ipynb

```diff
@@ -0,0 +1,995 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "qualified-subdivision",
+   "metadata": {},
+   "source": [
+    "# Cube Reprojection Tutorial\n",
+    "\n",
+    "This tutorial shows how to take two spectral cubes observed toward the same part of the sky, but different frequencies, and put them onto the same grid.\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "higher-console",
+   "metadata": {},
+   "source": [
+    "## Index \n",
+    "\n",
+    " * [Step 1: Download](#Step-1:-Download-the-data)\n",
+    " * [Step 2: Open files, collect metadata](#Step-2:-Load-the-cubes)\n",
+    " * [Step 3: Convert to velocity](#Step-3:-Convert-cubes-from-frequency-to-velocity)\n",
+    " * [Step 4: Spectral Interpolation](#Step-4.-Spectral-Interpolation)\n",
+    " * [Step 5: Spatial Smoothing](#Step-5.-Spatial-Smoothing)\n",
+    " * [Step 6: Reprojection](#Step-6.-Reprojection)\n",
+    " \n",
+    " \n",
+    "In this example, we do spectral smoothing and interpolation (step 4) before spatial smoothing and interpolation (step 5), but if you have a varying-resolution cube (with a different beam size for each channel), you have to do spatial smoothing first."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "fewer-london",
+   "metadata": {},
+   "source": [
+    "## Step 1: Download the data\n",
+    "\n",
+    "(you might not have to do this step, since you may already have data)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "id": "revised-upgrade",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import numpy as np\n",
+    "from astropy.utils.data import download_file"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "id": "eastern-three",
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "'/home/adamginsburg/.astropy/cache/download/url/33718a6f64d8a437fe89bff80278ca22/contents'"
+      ]
+     },
+     "execution_count": 2,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "filename_1 = download_file(\"https://almascience.nrao.edu/dataPortal/member.uid___A001_X1465_X3a33.BrickMaser_sci.spw71.cube.I.manual.image.pbcor.fits\",\n",
+    "                           cache=True)\n",
+    "filename_1"
```

Comment:
I'm unsure whether printing these odd-looking file names is useful or confusing.